OpenIGTLinkIF/MRML/vtkMRMLIGTLConnectorNode.cxx

```diff
@@ -883,6 +909,10 @@ void vtkMRMLIGTLConnectorNode::ProcessIOConnectorEvents(vtkObject *caller, unsig
         this->Internal->ProcessOutgoingDeviceModifiedEvent(caller, event, modifiedDevice);
         }
       }
+    else if (mrmlEvent == AboutToReceiveEvent)
```

Comment:
add igtlioDevice:: prefix

Reply:
In this case it would be vtkMRMLIGTLConnectorNode::AboutToReceiveEvent.

That made me think of something though:
Currently the AboutToReceiveEvent is mostly internal to the connector node, although I think it still invokes an AboutToReceive event.

Are these external events useful, or should I remove the external visibility of AboutToReceiveEvent?

Reply:
I think it should be internal.